web/packages/teleterm/src/ui/ThemeProvider/theme.ts

```diff
@@ -114,11 +115,38 @@ const borders = [
   '32px solid',
 ];
 
+function getSansSerif() {
+  const platform = getPlatform();
+
+  if (platform.isLinux) {
+    return "system-ui, 'Ubuntu', 'Droid Sans', sans-serif";
+  }
+
+  if (platform.isMac) {
+    return '-apple-system, BlinkMacSystemFont, sans-serif';
+  }
+
+  if (platform.isWin) {
+    return "system-ui, 'Segoe WPC', 'Segoe UI', sans-serif";
+  }
+}
+
+const sansSerif = getSansSerif();
```

Comment:
What's the reason for changing how the sans serif font is handled vs keeping it the way it was set?

Reply:
Do you mean moving it from `ConfigService`?

I think we should keep config as simple as possible. If something is not configurable, we can move it closer to the place it is actually used.
Also, we want to generate JSON schema for the config schema, so we would have to find a way to omit `fonts.sansSerif` (and now also `fonts.mono`).

Reply:
I mean not depending on `web/packages/design/src/theme/fonts.js`. As far as I can see, on master we read `fonts.sansSerif` from this file while this PR copies that here.

Reply:
Yees, we read the initial value from `fonts.js`, but the value is immediately replaced with the value from config: https://github.com/gravitational/teleport/blob/a15e987476a0cf78b84aa7751f0b0eee06ff3640/web/packages/teleterm/src/ui/ThemeProvider/ThemeProvider.tsx#L32 
And the sansSerif copied from `configService.ts` !== sansSerif from `fonts.js`.

I started wondering if we could just use `system-ui` for all systems in Connect. [It is supported](https://caniuse.com/font-family-system-ui) in Chrome form ages.

Reply:
I see. Totally agree that removing it from the config makes sense and I see why the change was made.

I'd say let's go with `system-ui` since as far as I understand, it'll help us limit all of the current sans-serif code to just one line.

Reply:
Done.